### PR TITLE
Document Windows Desktop 6.0.2 breaking change

### PR DESCRIPTION
**Pull request by RussKie**
Document the problem with Windows Desktop 6.0.2

Relates to
* https://github.com/dotnet/core/issues/7176
* https://github.com/dotnet/winforms/issues/6663
* https://github.com/dotnet/core/issues/7172#issuecomment-1034105137 
* https://github.com/dotnet/wpf/issues/6109 
* https://github.com/dotnet/wpf/issues/6111 
* https://github.com/dotnet/runtime/issues/65217 
* https://github.com/dotnet/sdk/issues/24240 
* https://github.com/dotnet/runtime/issues/67580

### CODE REVIEW

File: release-notes/6.0/known-issues.md [hunk 1]
```diff
@@ -58,3 +58,46 @@ To fix this, edit your `PATH` environment variable to either remove the `c:\Prog
...
+Description: The process was terminated due to an unhandled exception.
+Exception Info: System.IO.FileLoadException: Could not load file or assembly 'System.Windows.Forms, Version=6.0.2.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'. The located assembly's manifest definition does not match the assembly reference. (0x80131040)
+File name: 'System.Windows.Forms, Version=6.0.2.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'
+   at WinFormsApp1.Program.Main()   
+```
+
+This is a result of Windows Desktop servicing ref packs in 6.0.2, which was shipped with an incorrect version.
+
+**Fix:**
+* To run Windows Desktop applications built against 6.0.0 or 6.0.1, the Windows Desktop runtime either 6.0.0 or 6.0.1 must be installed.
```
Comment (dsplaisted) [inline release-notes/6.0/known-issues.md:80]: I think an app built against 6.0.0 or 6.0.1 should run correctly if 6.0.2 or higher is installed.

"Built against 6.0.2" isn't really precise either.  It's apps built with the 6.0.102 .NET SDK or higher.

Comment (RussKie) [reply release-notes/6.0/known-issues.md:80]: > I think an app built against 6.0.0 or 6.0.1 should run correctly if 6.0.2 or higher is installed.

Yes, you're absolutely correct. I just tested it on a brand new VM with only Windows Desktop runtime 6.0.3 installed, and was able to run a Windows Forms app built against 6.0.100.

Comment (RussKie) [reply release-notes/6.0/known-issues.md]: I made updates. How does it look now?
Thank you

File: release-notes/6.0/known-issues.md [hunk 2]
```diff
@@ -58,3 +58,45 @@ To fix this, edit your `PATH` environment variable to either remove the `c:\Prog
...
 
 When using localdb (default when creating projects in VS), the normal database apply migrations error page will not be displayed correctly due to the spa proxy. This will result in errors when going to the fetch data page. Apply the migrations via 'dotnet ef database update' to create the database.
+
+## Windows Desktop (Windows Forms / WPF)
+
+### Issues running applications with Windows Desktop 6.0.2
+
+Some customers are unable to run Windows Desktop (that is, Windows Forms or WPF) applications built with 6.0.200 or later .NET SDK, if the target workstation has only .NET Windows Desktop runtime 6.0.0 or 6.0.1 installed, and receieve error messages similar to the following:
```
Comment (dsplaisted) [inline release-notes/6.0/known-issues.md:66]: I'm not sure "workstation" is the right word here.

Comment (RussKie) [reply release-notes/6.0/known-issues.md:66]: Would this work better?
```suggestion
Some customers are unable to run Windows Desktop (that is, Windows Forms or WPF) applications built with 6.0.200 or later .NET SDK, if the target environment has only .NET Windows Desktop runtime 6.0.0 or 6.0.1 installed, and receive error messages similar to the following:
```

File: release-notes/6.0/known-issues.md [hunk 3]
```diff
@@ -58,3 +58,45 @@ To fix this, edit your `PATH` environment variable to either remove the `c:\Prog
...
+    {
+      "sdk": {
+        "version": "6.0.100",
+        "rollForward": "disable"
+      }
+    }
+    ```
+    ...or locking the runtime at the project level:
+    ```xml
+    <ItemGroup>
```
Comment (RussKie) [inline release-notes/6.0/known-issues.md:95]:
```suggestion
    <ItemGroup Condition="'$(TargetFrameworkVersion)' == '6.0'">
```

File: release-notes/6.0/known-issues.md [hunk 4]
```diff
@@ -58,3 +58,45 @@ To fix this, edit your `PATH` environment variable to either remove the `c:\Prog
...
+* If we lock the ref pack version number at 6.0.2, there is a workaround that allows building an app or library that can run on 6.0.0 or 6.0.1 - for the end-user it requires installing Windows Desktop runtime 6.0.0 or 6.0.1, and for a developer either locking the SDK in global.json:
+    ```json
+    {
+      "sdk": {
+        "version": "6.0.100",
+        "rollForward": "disable"
+      }
+    }
+    ```
+    ...or locking the runtime at the project level:
```
Comment (RussKie) [inline release-notes/6.0/known-issues.md:93]:
```suggestion
* If we lock the ref pack version number at 6.0.2, there is a workaround that allows building an app or library that can run on 6.0.0 or 6.0.1 - for an end-user it requires installing Windows Desktop runtime 6.0.0 or 6.0.1, and for a developer - locking the runtime at the project level:
```

**Comment by RussKie** [comment]
@dsplaisted may I get you to re-review again? Thank you.

**Review by dsplaisted — APPROVED** [review]

**Pull request merged**